test/integration/stamp_editor_spec.js

```diff
@@ -198,4 +199,158 @@ describe("Stamp Editor", () => {
       );
     });
   });
+
+  describe("Alt text dialog", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("empty.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that the alt-text flow is correctly implemented", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.click("#editorStamp");
+
+          const data = fs
+            .readFileSync(path.join(__dirname, "../images/firefox_logo.svg"))
+            .toString("base64");
+          await page.evaluate(async imageData => {
+            const resp = await fetch(`data:image/svg+xml;base64,${imageData}`);
+            const blob = await resp.blob();
+            window.PDFViewerApplication.eventBus.dispatch("pasteeditor", {
+              source: this,
+              details: {
+                type: 13,
```

Comment:
You don't need to hard-code this:
```suggestion
                type: window.pdfjsLib.AnnotationEditorType.STAMP,
```